tests/molecules/He_param/He_orb_rot_param_grad_legacy.xml

```diff
@@ -90,7 +90,7 @@
       <parameter name="steps"> 10 </parameter>
       <parameter name="substeps"> 20 </parameter>
       <parameter name="timestep"> 0.5 </parameter>
-      <parameter name="samples"> 1000 </parameter>
+      <parameter name="samplesperthread"> 1000 </parameter>
```

Comment:
Could you use the total number of samples and input it still as samples? samplesperthread was never a good input even with legacy drivers.